Generals/Code/GameEngine/Source/Common/RTS/TunnelTracker.cpp

```diff
@@ -353,6 +399,7 @@ void TunnelTracker::xfer( Xfer *xfer )
 
 		}
 
+		m_needsFullHealTimeUpdate = true;
```

Comment:
```cpp
if( xfer->getXferMode() == XFER_LOAD )
{
  m_needsFullHealTimeUpdate = true;
}
```
I think. Because when saving we do not need an update.

There already is such a branch in this function.

Reply:
Do we need to be explicit? It is already in the else block of the `XFER_SAVE` condition.
```cpp
if( xfer->getXferMode() == XFER_SAVE )
{
  // ...
}
else
{
  m_needsFullHealTimeUpdate = true;
}
```